exploits/minitwlpayload/bootloader/source/boot.c

```diff
@@ -56,6 +56,7 @@ void arm7clearRAM();
 //+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 // Important things
 #define TEMP_MEM 0x02FFE000
```

Comment:
This should be changed to 0x02FFD000 as per https://github.com/devkitPro/nds-bootloader/blob/master/source/boot.c#L62